### PR TITLE
Address some usability issues with service affinity rules

### PR DESCRIPTION
**Pull request by sonchang**

### CODE REVIEW

**Comment by sonchang** [comment]
@alena1108 @cjellick Can you review?  The drone TF looks unrelated

File: code/iaas/allocator/src/main/java/io/cattle/platform/allocator/service/AllocatorService.java [hunk 1]
```diff
@@ -14,6 +14,12 @@
      */
     void mergeLabels(Map<String, String> srcMap, Map<String, String> destMap);
 
+    /**
+     * This address various usability issues allowing users to provide shorthand versions of the service
```
Comment (cjellick) [inline code/iaas/allocator/src/main/java/io/cattle/platform/allocator/service/AllocatorService.java]: Typo: s/address/addresses

**Comment by alena1108** [comment]
LGTM

**Comment by cjellick** [comment]
LGTM

**Comment by cjellick** [comment]
Son confirmed the build failure is from unrelated failures. Merging.

**Pull request merged**